holistics/holistics.py

```diff
@@ -5,95 +5,96 @@
             "Content-Type":"application/json",
-            "X-Holistics-Key": None}
-    page={
-            'job_id': None,
+            "X-Holistics-Key": 'api_key'}
+        self.page={
+            'job_id': 'job_id',
             '_page_size': '10000000',
             '_page': '10000000'}
-
-    def __init__(self, api_key, path = None, url = None):
         self.headers['X-Holistics-Key'] = api_key
         if url is not None:
             self.url = url
-        if path is not None:
-            self.path = path
-
-    def info(self):
-        return {'api_key': self.headers['X-Holistics-Key'], 'path': self.path, 'url': self.url, }
+        
 
-    def GetURL(self, tail_url, params=None):
-        try:
-            res = requests.get(self.url + tail_url, params = params, headers = self.headers)
-            res.raise_for_status()
-        except requests.exceptions.HTTPError as err:
-            print (err)
-            return 0
+    def get_url(self, tail_url, params=None):
+        res = requests.get(self.url + tail_url, params = params, headers = self.headers)
+        res.raise_for_status()
         return res
-        
 
-    def SubmitReport (self, report_id, filters=None):
+    def submit_export (self, report_id, filters=None):
         print ("Submitting export request... ", end='')
         tail_url = '/queries/'+str(report_id)+'/submit_export.csv'
-        res = self.GetURL(tail_url, filters)
-        if res!=0:
+        try:
+            res = self.get_url(tail_url, filters)
             res = res.json()
             print ("Success")
             self.page['job_id'] = str(res['job_id'])
+            print (str(res['job_id']))
             return 1
-        else:
-            return 0
+        except requests.exceptions.HTTPError as err:
+            print ("Fail to submit export")
+            return err
 
-    def GetExportResults(self, job_id, _page_size = None, _page = None):
+    def get_export_results(self, job_id, _page_size = None, _page = None):
         print ("Getting export results... ", end='')
         tail_url = '/queries/get_export_results.json'
-        self.page['job_id']=job_id
+        self.page['job_id']=str(job_id)
         if _page_size is not None:
             self.page['_page_size']=_page_size
         if _page is not None:
             self.page['_page']=_page
-        res = self.GetURL(tail_url, self.page)
-        if res!=0:
+        try:
+            res = self.get_url(tail_url, self.page)
             res = res.json()
             while (res['status'] != 'success'):
                 if res['status'] == 'already_existed':
                     self.page['job_id'] = str(res['job_id'])
-                    break
                 if res['status'] == 'failure':
                     print ("Status: Failure")
-                    return 0
-                res = self.GetURL(tail_url, self.page).json()
+                    raise ValueError('Can\'t get export results')
```

Comment:
Incorrect usage of ValueError. Either create a new Exception class or use RuntimeError. 

Must also pump up the error message returned by the job result to let user knows what went wrong with the job. E.g.
```python
raise RuntimeError(res['error'])
```

Or at least print the `res['error']`